packages/waas/src/intents/transactions.ts

```diff
@@ -59,6 +60,16 @@ export type SendERC1155Args = TransactionFeeArgs & {
   data?: string
 }
 
+export type SendContractCallArgs = TransactionFeeArgs & {
+  chainId: number
+  to: string
+  value?: ethers.BigNumberish
+  abi: string
+  func?: string
```

Comment:
func is optional, as its only needed if you pass in the full abi object, if you pass just a function selector, then that implies the function from the selector